src_c/polygon.c

```diff
@@ -468,6 +468,31 @@ pg_polygon_move(pgPolygonObject *self, PyObject *const *args, Py_ssize_t nargs)
     return tmp;
 }
 
+static PyObject *
+pg_polygon_as_segments(pgPolygonObject *self) {
```

Comment:
`METH_NOARGS` functions should have a second parameter that's `_null`
```suggestion
pg_polygon_as_segments(pgPolygonObject *self, PyObject *_null) {
```